src/Calculator.cpp

```diff
@@ -0,0 +1,106 @@
+    if ((c >= '0') and (c <= '9')) {
+        return true;
+    }
+    else {
+        return false;
+    }
+}
+
+std::vector<std::string> str_to_strvec(std::string equ_str) {
+    std::string num_str;                                    //Variable to hold numeric characters, to combine multiple numerals into a single number-string
+    std::vector<std::string> equ_vec;                       //Variable to hold vector version of equation string
+    for (int i = 0; i < equ_str.length(); i++) {              //Iterate i over indices in equ_str vector
+        std::string character = { equ_str[i] };               //Define character variable as a string, and assign the string conversion of the entry of equ_str at index i
+
+        if (char_is_num(equ_str[i])) {                          //Check if the character in equ_str at index i is a number.
+            if (i == 0) {                                       //If so, if it is the first character in the string,
+                num_str = character;                            //or if it is the preceding character is not a number,
+            }                                                   //then it is the first character of a number-string,
+            else if (char_is_num(equ_str[i - 1]) == false) {    //and num_str is set equal to character, the string version of the character
+                num_str = character;
+            }
+            else {                                              //If the character is not the first character of a number-string, but is a number,
+                num_str.append(character);                      //append the string version to the end of the num_str string.
+            }
+
+            if (i == (equ_str.length() - 1)) {                  //If i is at the end of the vector,
+                equ_vec.push_back(num_str);                     //or the next character is not a number,
+            }                                                   //the current character is the last character in its number-string.
+            else if (char_is_num(equ_str[i + 1]) == false) {    //So, num_str is added to the end of the equ_vec vector
+                equ_vec.push_back(num_str);
+            }                                                   //If not, nothing is done.
+        }
+        else if (character == "+") {                            //If the current character is a +,
+            equ_vec.push_back(character);                       //add the string version to the end of equ_vec.
+        }
+        else {                                                  //If an unrecognized character is detected, print error message and abort loop
+            std::cout << "ERROR: Unrecognized input: " << character << std::endl;   // - needs implementation of actual error handling process
+            break;
+        }
+    }
+    return equ_vec;     //Return created vector
+}
+
+std::vector<std::string> add_sub_calc(std::vector<std::string> equ_vec) {

```

Comment:
Convert to reference (lets us actually modify equ_vec rather than modifying the copy)
```cpp
std::vector<std::string> add_sub_calc(std::vector<std::string> &equ_vec) {
```